sig-apps/agenda.md

```diff
@@ -1,80 +0,0 @@
-# Agenda
```

Comment:
maybe we should rather move the agenda and mintes than remove these? We could either move them to a separate doc like we do with our modern minutes or move all these files into the `archive` folder.